hello.go

```diff
@@ -75,6 +82,40 @@ func parseObjectHeader(bufScanner *bufio.Scanner) objectHeader {
 	return objectHeader{headerComponents[0], objectLen}
 }
 
+func printCommitContent(bufScanner *bufio.Scanner, byteCount int) {
+	//format:
+	// tree <tree sha>
+	// parent <parent sha>
+	// [parent <parent sha> if several parents from merges]
+	// author <author name> <author e-mail> <timestamp> <timezone>
+	// committer <author name> <author e-mail> <timestamp> <timezone>
+
+	// <commit message>
+
+	fileMetadataBytes := scanCountBytes(bufScanner, byteCount, true)
+	fileMetadataString := string(fileMetadataBytes)
+	fmt.Print(fileMetadataString)
+}
+
+func printBlobContent(bufScanner *bufio.Scanner, byteCount int) {
+	//format:
+	// <content>
+	// ...
+	// print 3KB size (atmax) of object content
+	var count int
+	if (byteCount > 3072) {
```

Comment:
declare `3072` as a constant at the top the file or function instead of duplicating at 3 places.